### PR TITLE
[docs] Fix verbiage issues in headless Tabs doc

### PR DESCRIPTION
**Pull request by amandeepmittal**
# Why

Add an example and fix verbiage issues in Create Tab layouts document.

Based on feedback from @keith-kurak, update the navigation structure of this doc.

# Test Plan

Run docs locally.

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/pages/router/ui/tabs.mdx [hunk 1]
```diff
@@ -35,25 +40,25 @@ Expo Router offers a set of components to create custom Tabs layouts. Unlike the
...
 
-The trigger `<Trigger name="dynamic page" href="/hello-world" />` will create a tab for `[slug].tsx` with the params `{ slug: 'hello-world' }`
+The trigger `<Trigger name="dynamic page" href="/hello-world" />` will create a tab for **[slug].tsx** with the params `{ slug: 'hello-world' }`.
 
 ### Ambagious routes
 
 <FileTree files={['_layout.tsx', ['(one,two)/route.tsx', 'A route within a shared group']]} />
 
-The `href` values provided to `<TabTrigger />` cannot be ambagious and must always point to a single route. The href `/route` is not allowed, as it could refer to either `/(one)/route` or `/(two)/route`.
+The `href` values provided to `<abTrigger` cannot be ambagious and must always point to a single route. The href `/route` is not allowed, as it could refer to either `/(one)/route` or `/(two)/route`.
```
Comment (entiendoNull) [inline docs/pages/router/ui/tabs.mdx:61]: `TabTrigger` instead of `<abTrigger`, right?

Comment (amandeepmittal) [reply docs/pages/router/ui/tabs.mdx:61]: Good catch 😅

File: docs/pages/router/advanced/custom-tabs.mdx [hunk 2]
```diff
@@ -0,0 +1,243 @@
+---
+title: Custom Tab layouts
+description: Learn how to use headless tab components to create custom tab layouts in Expo Router.
+sidebar_title: Custom tabs
+---
+
+import { Collapsible } from '~/ui/components/Collapsible';
+import { FileTree } from '~/ui/components/FileTree';
+
+> **warning** Experimentally available in SDK 52 and above. For the React Navigation styled tabs layout that are commonly used in native apps, see [Tabs](/router/advanced/tabs/).
```
Comment (keith-kurak) [inline docs/pages/router/advanced/custom-tabs.mdx:10]:
```suggestion
> **warning** Experimentally available in SDK 52 and above. For the React Navigation-styled tabs layout that are commonly used in native apps, see [Tabs](/router/advanced/tabs/).
```

File: docs/pages/router/advanced/custom-tabs.mdx [hunk 3]
```diff
@@ -0,0 +1,243 @@
+---
+title: Custom Tab layouts
```
Comment (keith-kurak) [inline docs/pages/router/advanced/custom-tabs.mdx:2]:
```suggestion
title: Custom tab layouts
```
We usually only capitalize first letter, unless it's a product name / proper noun, right?

Comment (amandeepmittal) [reply docs/pages/router/advanced/custom-tabs.mdx:2]: You are absolutely right. My bad. I'll get this fixed.

**Review by keith-kurak — APPROVED** [review]
Added everything I could put together quickly based on my experience demoing these. 

I didn't touch **ui/tabs.mdx**.. It seems like it repeats a lot from **custom-tabs.mdx**. So, you could adapt the same bits down there as applicable, but I'm also wondering if we could just delete it? I think it would be confusing to see both of these. This is an advanced concept and makes a lot of sense there.

**Review by keith-kurak — CHANGES_REQUESTED** [review]
Ah shoot that was supposed to be request changes, sorry

**Comment by amandeepmittal** [comment]
> Added everything I could put together quickly based on my experience demoing these.

Thank you so much!

> I didn't touch **ui/tabs.mdx**.. It seems like it repeats a lot from **custom-tabs.mdx**. So, you could adapt the same bits down there as applicable, but I'm also wondering if we could just delete it? I think it would be confusing to see both of these. This is an advanced concept and makes a lot of sense there.

Yeah that wasn't meant to be. Sorry about causing the confusion in the PR itself. I had some trouble resolving conflicts in this PR. For some unknown reason I wasn't able to resolve it (maybe due to the file was deleted in this PR and then there were some changes on the main). The page isn't visible in the docs. I'll create a follow up PR to remove `ui/tabs.mdx` as soon as this is shipped.

**Comment by keith-kurak** [comment]
> > Added everything I could put together quickly based on my experience demoing these.
> 
> Thank you so much!
> 
> > I didn't touch **ui/tabs.mdx**.. It seems like it repeats a lot from **custom-tabs.mdx**. So, you could adapt the same bits down there as applicable, but I'm also wondering if we could just delete it? I think it would be confusing to see both of these. This is an advanced concept and makes a lot of sense there.
> 
> Yeah that wasn't meant to be. Sorry about causing the confusion in the PR itself. I had some trouble resolving conflicts in this PR. For some unknown reason I wasn't able to resolve it (maybe due to the file was deleted in this PR and then there were some changes on the main). The page isn't visible in the docs. I'll create a follow up PR to remove `ui/tabs.mdx` as soon as this is shipped.

No worries, that makes sense and sounds good!

**Review by keith-kurak — APPROVED** [review]
shipit!

**Pull request merged**